private/clusterstat.m

```diff
@@ -326,7 +326,7 @@
       
       numdims = length(cfg.dim);
       if numdims == 2 || numdims == 3 % if 2D or 3D data
-        [negclusrnd, negrndnum] = spm_bwlabel(tmp, 2*numdims); % use spm_bwlabel for 2D/3D to avoid usage of image toolbox
+        [negclusrnd, negrndnum] = spm_bwlabel(tmp, 6); % use spm_bwlabel for 2D/3D to avoid usage of image toolbox
```

Comment:
Agreed. Changes are clear and make sense. Apparently - before - nobody ended up at this point with numdims being 2, and thus never got an error. Indeed it is clear that spm_bwlabel only supports 6 (and not 4) as an input value